core/src/main/java/org/apache/carbondata/core/scan/executor/impl/SearchModeVectorDetailQueryExecutor.java

```diff
@@ -56,6 +59,11 @@
     }
   }
 
+  public static void shutdown() {
```

Comment:
I don't find the caller method of this method.

Reply:
Better override the finish method to call the executorservice shutdowm

Reply:
This is called in Worker.shutdown